resource/modify_plan.go

```diff
@@ -65,4 +81,14 @@ type ModifyPlanResponse struct {
 	// indicates a successful plan modification with no warnings or errors
 	// generated.
 	Diagnostics diag.Diagnostics
+
+	// Deferred indicates that Terraform should defer
+	// importing this resource.
```

Comment:
Similar suggestion here:

```suggestion
	// Deferred indicates that Terraform should defer importing this
	// resource until a followup apply operation.
```